Gemfile

```diff
@@ -175,3 +175,5 @@ group :development, :production do
   # Semantic logger
   gem "rails_semantic_logger"
 end
+
+gem "ostruct", "~> 0.6.0"
```

Comment:
It might be nicer for this to be further to the top of this file – i.e. so it's above the group-specific gems. Right now it's sitting all on its lonesome at the bottom.

Reply:
Ah the joys of using bundler directly! Will do :)